specification/migrateprojects/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-migrateprojects
+  package-version: 1.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/migrateprojectsManagement
+  generate-license-txt: true
+  generate-package-json: true
+  generate-readme-md: true
```

Comment:
Please also create a readme.typescript.md file. You can find several other packages in this repository that have readme.typescript.md files that you can copy and modify slightly to get the results you need.